requirements/lint.txt

```diff
@@ -1,9 +1,8 @@
 -r pip.txt
-maxcdn
-# astroid 1.6.2 breaks pylint-django
-# https://github.com/PyCQA/pylint-django/issues/117
-astroid==1.6.1
-pylint
-prospector
-pylint-django
-pyflakes
+maxcdn==0.0.7
```

Comment:
Thi dependency is also on `deploy.txt`, and if it's deleted all tests pass, not sure if we really need this here.

Reply:
Yea, we aren't supporting maxcdn anymore, but we might have tests for it. This is related to #2091